packages/compass-import-export/src/import/import-csv.spec.ts

```diff
@@ -108,7 +109,7 @@ describe('importCSV', function () {
           ignoreEmptyStrings: true,
         });
 
-        expect(result).to.deep.equal({
+        expect(omit(result, 'biggestDocSize')).to.deep.equal({
```

Comment:
In this test suite and also the import-json test suite, I am omitting `biggestDocSize` from the expected result because I test that explicitly in a different suite